main.py

```diff
@@ -799,10 +531,6 @@ def _send_flow_mods_from_request(self, dpid, command, flows_dict=None):
                 raise BadRequest(result)
 
         force = bool(flows_dict.get("force", False))
-        log.info(
-            f"Send FlowMod from request dpid: {dpid}, command: {command}, "
-            f"force: {force}, flows_dict: {flows_dict}"
-        )
```

Comment:
Amazing and also surprising the benchmark you presented, @viniarck. It is good to know about this because we can certainly remove some `log.debug` from the code then. I agree with you and we can keep the log.info as it is, and also add it to the handler `kytos.flow_manager.flows.(install|delete)`. Indeed, since it does not add that much, we can leave it to 2022.3 or so.

Reply:
> > Very nice to have mongodb for flow_manager and it is very satisfying to see the performance improvements enabled by this PR. I've reviewed the code and also run a few tests and it looks good to me!
> > I've tested this PR in different ways:
> > 
> > * Running manual tests to understand what is going on step by step
> > * Running the migration scripts/procedure
> > * Running end-to-end tests ([found a minor issue in the end-to-end related to ordering after enabling some tests for sdntrace/sdntrace-cp](https://github.com/amlight/kytos-end-to-end-tests/issues/134))
> > * restarting Kytos multiple times and checking the switches's flow table wouldn't be affected
> > * deleting flows from switches's flow tables and checking if kytos would recreate then
> > * etc.
> > 
> > My only comment would regarding the removal of `log.info` from the hot path of send_flow_mods. I think that logging is quite useful for troubleshooting and actually understand what is going on in the application (especially when you have to do that a few hours later an event). I know we are going towards having ES, APM, etc. in place, but having the log messages usually pays off when you are trying to make sense out of a problem (old school troubleshooting style). So that, I suggest include the `log.info` back to send_flow_mods, unless there is a huge impact and we cannot improve other ways.
> > Once again, congratulations!
> 
> @italovalcy I really appreciated your review and thanks for also exploring this as well, good to know. I'll check it out this e2e test that you mentioned to better understand. Regarding the log.info, I've replied to you on another thread.

very well, @viniarck! In regards to the e2e tests, I end up detecting that the test was broken due to the sdntrace/sdntrace_cp changes (fixed on commit [c2eb65](https://github.com/amlight/kytos-end-to-end-tests/pull/133/commits/c2eb65d6705d449c26776800905df491bc9d70e2)).

Reply:
Good to know, @italovalcy about this e2e test. Also, I've opened a new PR for this log info entry on that handler